validation/src/main/scala/hmda/validation/rules/lar/validity/V290.scala

```diff
@@ -1,36 +1,19 @@
 package hmda.validation.rules.lar.validity
 
-import hmda.model.census.CBSATractLookup
 import hmda.model.fi.lar.LoanApplicationRegister
 import hmda.validation.dsl.Result
 import hmda.validation.rules.EditCheck
 import hmda.validation.dsl.PredicateCommon._
 import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.PredicateGeo._
 
 object V290 extends EditCheck[LoanApplicationRegister] {
 
-  val cbsaTracts = CBSATractLookup.values
-
-  val validCombinations = cbsaTracts.map { cbsa =>
-    (cbsa.state, cbsa.county, cbsa.geoIdMsa)
-  }.toSet
-
-  val validMdCombinations = cbsaTracts.map { cbsa =>
-    (cbsa.state, cbsa.county, cbsa.metDivFp)
-  }.toSet
-
```

Comment:
I very much like that you moved this stuff out of here, and into a shared place with similar code.